diff -up libthai-0.1.9/doc/Doxyfile.in.doxygen-segfault libthai-0.1.9/doc/Doxyfile.in
--- libthai-0.1.9/doc/Doxyfile.in.doxygen-segfault	2007-11-13 16:37:16.000000000 -0500
+++ libthai-0.1.9/doc/Doxyfile.in	2007-11-13 16:37:31.000000000 -0500
@@ -29,7 +29,6 @@ STRIP_FROM_PATH        = @top_srcdir@/in
 STRIP_FROM_INC_PATH    = @top_srcdir@/include
 SHORT_NAMES            = NO
 JAVADOC_AUTOBRIEF      = NO
-QT_AUTOBRIEF           = NO
 MULTILINE_CPP_IS_BRIEF = NO
 DETAILS_AT_TOP         = YES
 INHERIT_DOCS           = YES
@@ -50,7 +49,6 @@ EXTRACT_PRIVATE        = NO
 EXTRACT_STATIC         = NO
 EXTRACT_LOCAL_CLASSES  = YES
 EXTRACT_LOCAL_METHODS  = NO
-EXTRACT_ANON_NSPACES   = NO
 HIDE_UNDOC_MEMBERS     = YES
 HIDE_UNDOC_CLASSES     = YES
 HIDE_FRIEND_COMPOUNDS  = NO
@@ -168,7 +166,6 @@ HTML_FOOTER            = 
 HTML_STYLESHEET        = 
 HTML_ALIGN_MEMBERS     = YES
 GENERATE_HTMLHELP      = NO
-HTML_DYNAMIC_SECTIONS  = NO
 CHM_FILE               = 
 HHC_LOCATION           = 
 GENERATE_CHI           = NO
